front/lib/api/assistant/global_agents.ts

```diff
@@ -969,26 +941,31 @@ The assistant always respects the mardown format and generates spaces to nest co
     4. If the user's query require neither internal company data or recent public knowledge, the assistant is allowed to answer without using any tool.
     The assistant always respects the mardown format and generates spaces to nest content.`;
 
-    dataSources.forEach((ds) => {
-      if (ds.connectorProvider && ds.connectorProvider !== "webcrawler") {
+    dataSourceViews.forEach((dsView) => {
```

Comment:
Make sure to flag in the PR description that this will break all global agents locally.